samcli/commands/sync/command.py

```diff
@@ -77,15 +77,15 @@
 """
 
 DESCRIPTION = """
-  By default, `$sam sync` runs a full AWS Cloudformation stack update.
+  By default, `$ sam sync` runs a full AWS Cloudformation stack update.
```

Comment:
I feel like all of these that are inline on a longer paragraph shouldn't even have the `$` in the first place (like the "`sam sync --watch`" in the following line), but it can be left for a future PR